packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -163,6 +166,7 @@ class Dialog extends AutoControlledComponent<WithAsProp<DialogProps>, DialogStat
   getInitialAutoControlledState(): DialogState {
     return {
       open: false,
+      documentRef: documentRef.current as HTMLDocument,
```

Comment:
I don't see from where this is captured

Reply:
I am using it from `import { documentRef } from '@fluentui/react-component-event-listener'`

Reply:
It should be taken from `this.context` (`this.content.target` is an intance of `document`) otherwise it will not for multiple windows

Reply:
Thanks... Would it be `props.content.content`? 🤔

Reply:
Done